CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.11.1
+
+  - Fix facet cache on document update (#789) 
+  - Improvements on settings consistency (#778)
```

Comment:
I don't see the #735 here or in the v0.11.1 Milestones either. It's in v0.12.0 Milestones but this PR is already merged on master...

Reply:
Oh ok, I see, it's already in the v0.11.0

Reply:
it will be documented as part of the 0.12.0 :)